src/status_im/events.cljs

```diff
@@ -273,6 +273,51 @@
  (fn [cofx _]
    (hardwallet/navigate-to-authentication-method cofx)))
 
+(handlers/register-handler-fx
+ :accounts.create.ui/intro-wizard
+ (fn [cofx _]
+   (accounts.create/intro-wizard cofx)))
```

Comment:
there is no need to register handlers , you can use {:events []} in fx/defn , please move all these handlers in core , thx

Reply:
Done